Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -1270,9 +1269,8 @@ static void dispatchSyntheticMouseMove(LocalFrame& mainFrame, const WebCore::Flo
     };
     if (auto range = std::exchange(m_rangeForDropSnapshot, std::nullopt)) {
         if (auto textIndicator = TextIndicator::createWithRange(*range, defaultTextIndicatorOptionsForEditDrag, TextIndicatorPresentationTransition::None, { }))
-            textIndicatorData = textIndicator->data();
```

Comment:
something does not look right here

Reply:
So, I'm open to other options, but this basically used to make the indicator, and only if that succeeded, set the data, which was declared above.
We want to make the indicator, and set it, but since we're inside an if, I still pre-declare it because we still need to send an empty one (which is what happened before) if there's no range.